ministryofjustice/laa-apply-for-criminal-legal-aid
# Bump schemas version

## Description of change
And use fixture from the gem, instead of duplicating it.

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-217

## How to manually test the feature
NOTE: the datastore will stop accepting v0.1 documents and will force 1.0 with all the expected attributes as per schema:
https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas/blob/main/schemas/1.0/application.json

Current feature-flagged code will no longer work I suppose, but it is not worth it update it now, we can wait until we start pushing v1.0 documents again.

Old v0.1 in local datastore will also fail. Probably best to wipe local datastore and start with fresh documents.
